src/content/app/tools/blast/views/blast-form/BlastForm.tsx

```diff
@@ -34,6 +34,11 @@ import BlastSpeciesSelector from 'src/content/app/tools/blast/components/blast-s
 
 import styles from './BlastForm.scss';
 
+// ToDo: Remove the 2 imports below once PR is approved and revert the testConfig props change on line 53 (Only for development env and review env)
+// Then we need to feedback to backend to update the config file for the tools API and coordinate the master branch update
+import testConfig from 'tests/fixtures/blast/blastSettingsConfig.json';
+import { BlastSettingsConfig } from '../../types/blastSettings';
```

Comment:
Import order is not right here

Reply:
@imransl thanks for reviewing but the changes to this file is only temporary so that we can test using the local file. Once this PR is approved, the changes will be reverted for this file. Hope thats ok.